sdk/avs/Azure.ResourceManager.Avs/api/Azure.ResourceManager.Avs.netstandard2.0.cs

```diff
@@ -1923,6 +2113,24 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         public override string ToString() { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct DnsZoneType : System.IEquatable<Azure.ResourceManager.Avs.Models.DnsZoneType>
```

Comment:
this is very likely to collide with types in other RPs (for example `dns`), could we rename this to `AvsDnsZoneType`? Or you could add some more prefix to specify this.